examples/vault-auto-unseal/main.tf

```diff
@@ -66,8 +70,9 @@ data "template_file" "user_data_vault_cluster" {
     consul_cluster_tag_key   = "${var.consul_cluster_tag_key}"
     consul_cluster_tag_value = "${var.consul_cluster_name}"
 
-    kms_key_id = "${var.auto_unseal_kms_key_id}"
-    aws_region = "${data.aws_region.current.name}"
+    kms_key_id                   = "${data.aws_kms_alias.vault-example.target_key_id}"
+    aws_region                   = "${data.aws_region.current.name}"
+    vault_enterprise_license_key = "${var.vault_enterprise_license_key}"
```

Comment:
Is the license key itself considered a secret? i.e., do we need to tell users to encrypt it for production use cases? Of course, this is a bit of a chicken-and-egg, as Vault itself is what we'd generally use to manage secrets...

Reply:
Following discussion with Josh we decided to treat it with moderate secrecy and not encrypt it